docker/wms/README.md

```diff
@@ -0,0 +1,32 @@
+# WMS Customizations
```

Comment:
Just an idle thought ... at some point we could update our Confluence docs on GSC with links to READMEs like this.

Reply:
Agreed